daemon/mgr/container_validation.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/alibaba/pouch/daemon/logger/syslog"
 	"github.com/alibaba/pouch/pkg/system"
 
+	"github.com/alibaba/pouch/daemon/logger/jsonfile"
```

Comment:
Please remove the blank line before this import line and add a blank line below it.